backend/spec/features/admin/orders/shipments_spec.rb

```diff
@@ -37,6 +37,15 @@
       expect(page).to have_content("shipped package")
       expect(order.reload.shipment_state).to eq("shipped")
     end
+
+    it "doesn't send a shipping confirmation email when ask to suppress the mailer" do
+      find("#send_mailer").set(false)
+      
+      expect {
+        find(".ship-shipment-button").click
+        wait_for_ajax
```

Comment:
I would like to see the `wait_for_ajax` eliminated. We should expect something on the page to happen.